KASM Containers

justinglock40 opened the pull request.

justinglock40 committed:
Create kasmworkspace

justinglock40 committed:
Delete kasmworkspace

justinglock40 committed:
placeholder

justinglock40 committed:
Add files via upload

justinglock40 committed:
Delete placeholder

doob187 commented on apps/kasmworkspace/chrome.yml:
```diff
@@ -0,0 +1,42 @@
+      - 'TZ=${TZ}'
+      - 'UMASK=${UMASK}'
+      - 'VNC_RESOLUTION=1920x1080'
+      - 'VNC_PW=password'
```
No Hardcoded Value

doob187 commented on apps/kasmworkspace/discord.yml:
```diff
@@ -0,0 +1,42 @@
+      - 'TZ=${TZ}'
+      - 'UMASK=${UMASK}'
+      - 'VNC_RESOLUTION=1920x1080'
+      - 'VNC_PW=password'
```
Same.here

doob187 commented on apps/kasmworkspace/discord.yml:
```diff
@@ -0,0 +1,42 @@
+    networks:
+      - proxy
+    ports:
+      - '6905:6901'  
```
BLOCKPORT missed

doob187 commented on apps/kasmworkspace/chrome.yml:
```diff
@@ -0,0 +1,42 @@
+    networks:
+      - proxy
+    ports:
+      - '6902:6901'  
```
BLOCKPORT missed

doob187 commented on apps/kasmworkspace/discord.yml:
```diff
@@ -0,0 +1,42 @@
+    volumes:
+      - '/etc/localtime:/etc/localtime:ro'
+      - '${APPFOLDER}/discord:/config:rw'
+      - '/mnt:/mnt'
```
Use unionfs Local persistent mode

doob187 commented on apps/kasmworkspace/firefox.yml:
```diff
@@ -0,0 +1,42 @@
+    networks:
+      - proxy
+    ports:
+      - '6906:6901'  
```
BLOCKPORT again

doob187 commented on apps/kasmworkspace/firefox.yml:
```diff
@@ -0,0 +1,42 @@
+    volumes:
+      - '/etc/localtime:/etc/localtime:ro'
+      - '${APPFOLDER}/firefox:/config:rw'
+      - '/mnt:/mnt'
```
Same here use Local persistent

doob187 commented on apps/kasmworkspace/onlyoffice.yml:
```diff
@@ -0,0 +1,42 @@
+      - 'TZ=${TZ}'
+      - 'UMASK=${UMASK}'
+      - 'VNC_RESOLUTION=1920x1080'
+      - 'VNC_PW=password'
```
Password need value in env

doob187 commented on apps/kasmworkspace/onlyoffice.yml:
```diff
@@ -0,0 +1,42 @@
+    networks:
+      - proxy
+    ports:
+      - '6909:6901'  
```
BLOCKPORT  again

doob187 commented on apps/kasmworkspace/signal.yml:
```diff
@@ -0,0 +1,42 @@
+      - 'TZ=${TZ}'
+      - 'UMASK=${UMASK}'
+      - 'VNC_RESOLUTION=1920x1080'
+      - 'VNC_PW=password'
```
Value again

doob187 commented on apps/kasmworkspace/signal.yml:
```diff
@@ -0,0 +1,42 @@
+    networks:
+      - proxy
+    ports:
+      - '6904:6901'  
```
BLOCKPORT again

doob187 commented on apps/kasmworkspace/steam.yml:
```diff
@@ -0,0 +1,42 @@
+      - 'UMASK=${UMASK}'
+      - 'VNC_RESOLUTION=1920x1080'
+      - 'VNC_PW=password'
+    networks:
```
Env again

doob187 commented on apps/kasmworkspace/steam.yml:
```diff
@@ -0,0 +1,42 @@
+    networks:
+      - proxy
+    ports:
+      - '6903:6901'  
```
BLOCKPORT

doob187 commented on apps/kasmworkspace/telegram.yml:
```diff
@@ -0,0 +1,42 @@
+      - 'TZ=${TZ}'
+      - 'UMASK=${UMASK}'
+      - 'VNC_RESOLUTION=1920x1080'
+      - 'VNC_PW=password'
```
Emv

doob187 commented on apps/kasmworkspace/telegram.yml:
```diff
@@ -0,0 +1,42 @@
+    volumes:
+      - '/etc/localtime:/etc/localtime:ro'
+      - '${APPFOLDER}/telegram:/config:rw'
+      - '/mnt:/mnt'
```
Local persistent

doob187 commented on apps/kasmworkspace/telegram.yml:
```diff
@@ -0,0 +1,42 @@
+    networks:
+      - proxy
+    ports:
+      - '6905:6901'  
```
BLOCKPORT

doob187 commented on apps/kasmworkspace/tor.yml:
```diff
@@ -0,0 +1,42 @@
+      - 'TZ=${TZ}'
+      - 'UMASK=${UMASK}'
+      - 'VNC_RESOLUTION=1920x1080'
+      - 'VNC_PW=password'
```
Env

doob187 commented on apps/kasmworkspace/tor.yml:
```diff
@@ -0,0 +1,42 @@
+    networks:
+      - proxy
+    ports:
+      - '6908:6901'  
```
BLOCKPORT

doob187 commented on apps/kasmworkspace/tor.yml:
```diff
@@ -0,0 +1,42 @@
+    volumes:
+      - '/etc/localtime:/etc/localtime:ro'
+      - '${APPFOLDER}/tor:/config:rw'
+      - '/mnt:/mnt'
```
Local persistent

doob187 commented on apps/kasmworkspace/vlc.yml:
```diff
@@ -0,0 +1,42 @@
+      - 'TZ=${TZ}'
+      - 'UMASK=${UMASK}'
+      - 'VNC_RESOLUTION=1920x1080'
+      - 'VNC_PW=password'
```
Env

doob187 commented on apps/kasmworkspace/vlc.yml:
```diff
@@ -0,0 +1,42 @@
+    networks:
+      - proxy
+    ports:
+      - '6907:6901'  
```
BLOCKPORT

doob187 commented on apps/kasmworkspace/vlc.yml:
```diff
@@ -0,0 +1,42 @@
+    volumes:
+      - '/etc/localtime:/etc/localtime:ro'
+      - '${APPFOLDER}/vlc:/config:rw'
+      - '/mnt:/mnt'
```
Local persistent

doob187 reviewed (changes requested):
Fix PLZ

justinglock40 committed:
Update envmigrate.sh

add KASM Resolution Variable for 1080p

justinglock40 committed:
Delete kasmweb.yml

Moved to /apps/kasmworkspace

justinglock40 committed:
Config Fixes/Correction

Added Resolution/Password to ENV, Added PORTBLOCK, Switch volume /mnt to persistent storage (unionfs).

justinglock40 committed:
missing mountpoint

add unionfs mountpoint

doob187 reviewed (approved):
👍 from.my side

@drag0n141

drag0n141 reviewed (approved):
Looks good :)
